### PR TITLE
Added shell script to add files from staging to commit to push issue #56

### PR DESCRIPTION
**Pull request by VipulChauhan89**
I have implemented the script to allow the end user to choose the URL, commit message, and files to be staged. The script now provides prompts for these inputs, enabling customization based on user preferences. Thank you for assigning me this task, and please let me know if you have any further questions or if there's anything else I can assist you with.

closes issue #56

### CODE REVIEW

**Comment by sahil-sagwekar2652** [comment]
Please use "closes" or "fixes" keyword to auto-close the issues. I've already said that in the CONTRIBUTING.md

File: scripts/git_stage_commit_push.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# Prompt the user to enter the repository URL
+read -p "Enter the repository URL: " repo_url
+
+# Clone the repository with temp_repo name
+git clone $repo_url temp_repo
+
+# Change to the repository directory
+cd temp_repo
+
+# Prompt the user to enter the file names to be staged
```
Comment (sahil-sagwekar2652) [inline scripts/git_stage_commit_push.sh:12]: How are the files being staged when the user hasn't made any changes to them?

**Review by sahil-sagwekar2652 — CHANGES_REQUESTED** [review]
Check my comment in the code

**Review by VipulChauhan89 — COMMENTED** [review]
Now this code will first move the file from the file path to the temp_repo and this will override the files if it is already there in the folder after that, all the git flow will work from adding files to the staging area to commit to push the file to the global repository.

**Comment by VipulChauhan89** [comment]
Please have a look at my latest commit that is with that will clear your doubt of yours.
Commit id =75247f6d9ae19b23ab6c53c0118bf40ebc508fde

@sahil-sagwekar2652

**Pull request closed without merging**